gtests/test_nopython_radixtree.cpp

```diff
@@ -0,0 +1,78 @@
+#include <modmesh/toggle/RadixTree.hpp>
+#include <gtest/gtest.h>
+
+#ifdef Py_PYTHON_H
+#error "Python.h should not be included."
+#endif
+
+template <typename T>
+std::string get_info(modmesh::RadixTreeNode<T> & r)
```

Comment:
Helper for printing the contents.

Reply:
Keep in mind that the RadixTree is still a prototype.  It does not work for the profiling and stack printing yet.  It does not have a Python wrapper yet.  The usability cannot be defined yet.  It is too early to evaluate how users will use the information.  Because we do not have the information or experience for this feature, we do not add fragile code.

Adding fragile code in the tests is less intrusive, so we are doing it.